iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -1038,6 +1050,28 @@ static String encodeString(String rawString) {
     return HiveConf.EncoderDecoderFactory.URL_ENCODER_DECODER.encode(rawString);
   }
 
+  static String getLocation(String location, String defaultLocation, Configuration conf) {
+    if (location == null || defaultLocation == null) {
+      return location;
```

Comment:
If location is null or defaultLocation is null would not be better to throw an exception? Or it's value checked somewhere else?

Reply:
We are typically attempting to mask the location here, if the location is null, we can't mask the default location or there is nothing to mask in the location, if the Client considers it invalid it will throw during auth